bin/stacks/api-stack.ts

```diff
@@ -283,6 +308,16 @@ export class APIStack extends cdk.Stack {
     });
     quote.addMethod('POST', quoteLambdaIntegration);
 
+    const switchLambdaIntegration = new aws_apigateway.LambdaIntegration(switchLambdaAlias, {});
+    const switchResource = api.root.addResource('synthetic-switch', {
+      defaultCorsPreflightOptions: {
+        allowOrigins: aws_apigateway.Cors.ALL_ORIGINS,
+        allowMethods: aws_apigateway.Cors.ALL_METHODS,
```

Comment:
we should only allow URA to call this endpoint right? or else people will be able to sniff out whats enabled / not

Reply:
not CORS related but we can setup an internal api key
=

Reply:
added auth key and made it required for `enabled` endpoint